plugin/lib/src/main/java/com/snap/dagger/DaggerGraph.kt

```diff
@@ -29,7 +29,8 @@ data class DaggerBinding(
     val component: String,
     val module: String?,
     val scope: String?,
-    val dependencies: List<DaggerDependencyRequest>
+    val dependencies: List<DaggerDependencyRequest>,
+    val adjacentNodes: List<String>?
```

Comment:
nit: we should start adding documentation to these fields so that it's clear what they are.

Reply:
I think it is intentional that we didn't add documentation. All of them are direct api's of dagger's `BindingGraph`. But some kind of link to the original api definitions would be helpful.